asn1ate/sema.py

```diff
@@ -163,7 +163,7 @@ def strongconnect(node):
             component = tuple(connected_component)
             result.append(component)
 
-    for node in graph:
+    for node in sorted(graph):
```

Comment:
Could you use `graph.keys()` explicitly here? I've always found it subtle that `__iter__` for `dict` gets its keys, and together with `sorted` it took me a while to decode.